### DISCUSSION TITLE
Fresh Install - No traffic being seen

### DISCUSSION THREAD

**Discussion by drv2e30** [Security-Onion-Solutions/securityonion#2550] [General] [answered]
Hello All,

I was able to setup a fresh install on a pc using the iso for 2.4.3 on a machine that has plenty of space and power. I used the default options to include eval setup and airgap settings. Once it boots I make sure all services are up and functional, no issues. I then attached a separate machine to the management port and I can see the GUI and login without any issues. However I don't see any traffic passing on the system. 

I have the correct port attached to a SPAN port on a switch that I have tested with another pc that has wireshark and it's seeing traffic passing from the port. 

Any quick tips to check on?

**Comment by weslambert** [comment]
You'll want to first verify traffic is actually being seen on the monitor interface, using something like tcpdump.

**Comment by drv2e30** [comment, marked as answer]
Tried that, didn't see any traffic. I disconnected the NIC and connected
the network cable to another machine and traffic could be seen. Though once
I reconnected the network cable to the NIC it started to receive data!
Thanks for the idea!
